tokio-console/src/view/task.rs

```diff
@@ -127,15 +139,22 @@ impl TaskView {
             )
         };
 
+        let stats_constraints = if location_lines_vector.len() > 1 {
+            let area_needed_to_render_location = task.location().len() as u16;
+            [
+                // 15 is the length of "| Location:     |"
+                layout::Constraint::Min(area_needed_to_render_location + 15),
```

Comment:
Variable names are a bit long still. (you can hit "apply suggestion" to get the change directly applied)

```suggestion
            let location_len = task.location().len() as u16;
            [
                // 15 is the length of "| Location:     |"
                layout::Constraint::Min(location_len + 15),
```